src/main/java/com/marklogic/mapreduce/ContentWriter.java

```diff
@@ -544,6 +540,18 @@ protected void insertBatch(Content[] batch, int id)
                     // not retryable so trying to connect to the replica
                     LOG.warn(getFormattedBatchId() +
                         "RequestServerException: " + e.getMessage());
+                } else if (e instanceof ServerConnectionException) {
+                    // Bug:55607 - When the # of requests per connection reaches
+                    // the limit of Azure application gateway, inform the user
+                    // properly and stop retrying.
+                    if (e.getMessage().contains(
+                        "Server issued connection close within a batch")) {
+                        LOG.warn(getFormattedBatchId() +
```

Comment:
How many of these messages will customers roughly see per mlcp job?

Reply:
Depends on how many batches there are...One error message per batch